apps/admin-ui/src/realm-settings/NewAttributeSettings.tsx

```diff
@@ -23,13 +23,30 @@ import { useAlerts } from "../components/alert/Alerts";
 import { UserProfileProvider } from "./user-profile/UserProfileContext";
 import type { UserProfileAttribute } from "@keycloak/keycloak-admin-client/lib/defs/userProfileConfig";
 import type { AttributeParams } from "./routes/Attribute";
-import type { KeyValueType } from "../components/key-value-form/key-value-convert";
 import { convertToFormValues } from "../util";
 import { flatten } from "flat";
 
 import "./realm-settings-section.css";
 
-type UserProfileAttributeType = UserProfileAttribute & Attribute & Permission;
+type IndexedAnnotationsType = {
```

Comment:
Can we drop the `Type` suffix? It's kind of an anti-pattern, unless it's really needed.

```suggestion
type IndexedAnnotations = {
````